Production/python/TauL2Prod.py

```diff
@@ -3,22 +3,12 @@
 from Configuration.Eras.Era_Run3_cff import Run3
 import os
 
-txtList = {
-    "DY":["DYToLL_M-50_TuneCP5_14TeV-pythia8.txt"],
-    "QCD":["QCD_Pt-15to3000_TuneCP5_Flat_14TeV_pythia8.txt","QCD_Pt-15to7000_TuneCP5_Flat_14TeV_pythia8.txt","QCD_Pt-15to7000_TuneCP5_Flat_14TeV_pythia8_ext.txt","QCD_Pt_120to170_TuneCP5_14TeV_pythia8.txt","QCD_Pt_170to300_TuneCP5_14TeV_pythia8.txt","QCD_Pt_300to470_TuneCP5_14TeV_pythia8.txt","QCD_Pt_30to50_TuneCP5_14TeV_pythia8.txt","QCD_Pt_470to600_TuneCP5_14TeV_pythia8.txt","QCD_Pt_50to80_TuneCP5_14TeV_pythia8.txt","QCD_Pt_600oInf_TuneCP5_14TeV_pythia8.txt","QCD_Pt_80to120_TuneCP5_14TeV_pythia8.txt"],
-    "TT":["TTToSemiLeptonic_TuneCP5_14TeV-powheg-pythia8.txt","TT_TuneCP5_14TeV-powheg-pythia8.txt","TT_TuneCP5_14TeV-powheg-pythia8_ext.txt"],
-    "VBF":["VBFHToTauTau_M125_TuneCUETP8M1_14TeV_powheg_pythia80.txt"],
-    "WJets":["WJetsToLNu_TuneCP5_14TeV-amcatnloFXFX-pythia8.txt"],
-    "ZPrime":["ZprimeToTauTau_M-4000_TuneCP5_14TeV-pythia8-tauola.txt"]
-}
 
 options = VarParsing('analysis')
 options.register('sampleType', '', VarParsing.multiplicity.singleton, VarParsing.varType.string,
                  "Indicates the sample type: Summer16MC, Run2016, ...")
-options.register('txtList', '', VarParsing.multiplicity.singleton, VarParsing.varType.string,
-                 "List of txt of list of root files to process.")
-options.register('txtListPrefix', '', VarParsing.multiplicity.singleton, VarParsing.varType.string,
-                 "Directory of list of txt of list of root files to process.")
+options.register('inputFile', '', VarParsing.multiplicity.singleton, VarParsing.varType.string,
+                 "File to process")
```

Comment:
`inputFiles` is already in the list of default args for `analysis` preset, don't need to have `inputFile`